forum/forum-post.go

```diff
@@ -16,14 +16,19 @@ type NotificationGroups struct {
 	NotifyTo NotifyToType       `bson:"notify_to,omitempty" json:"notify_to,omitempty"`
 }
 
+type NotifyForum struct {
+	StudentList []primitive.ObjectID `bson:"student_list,omitempty"`
+	GroupsForum []primitive.ObjectID `bson:"groups_forum,omitempty"`
```

Comment:
Change to `groups` and should be `[]string` not `[]primitive.ObjectID` @shalearkane 

The reason is that it will be `cse_idd`, `mnc_idd` ....
Not any other collection

Reply:
alright